### PR TITLE
Feature/114 axn params template

### PR DESCRIPTION
**Pull request by crbrox**
Implements #114 

Add string substitution for some action parameters

@fgalan 
@dmoranj

### CODE REVIEW

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -1 +1,3 @@
 - Modify Perseo's executable to add environment variable-based configuration.
+- Add string subsitution for more action parameters
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE]: Given that in this case we have an issue for that feature (#114), let's incluce it in the CNR entry.

File: documentation/plain_rules.md [hunk 2]
```diff
@@ -106,7 +106,7 @@ Sends a SMS to a number set as an action paramter with the body of the message b
 ```
 The field `parameters` include a field `to` with the number to send the message to.
 
-The `template` field performs [attribute substitution](#string-substitution-syntax).
+The `template` and `to` fields perform [attribute substitution](#string-substitution-syntax).
```
Comment (fgalan) [inline documentation/plain_rules.md]: I understand that to keep documentation coherent, the list just under "This substitution can be used in the the following fields" at "String substitution syntax" should be also modified.

File: documentation/plain_rules.md [hunk 3]
```diff
@@ -183,7 +183,7 @@ Note that you can encode a JSON in the `template` field:
     }
 ```
 
-The `template` field performs [string substitution](#string-substitution-syntax).
+The `template` and `url` fields perform [string substitution](#string-substitution-syntax).
```
Comment (fgalan) [inline documentation/plain_rules.md]: I understand that to keep documentation coherent, the list just under "This substitution can be used in the the following fields" at "String substitution syntax" should be also modified.

File: documentation/plain_rules.md [hunk 4]
```diff
@@ -124,7 +124,7 @@ Sends an email to the recipient set in the action parameters, with the body mail
     }
 ```
 
-The `template` field performs [string substitution](#string-substitution-syntax).
+The `template`, `from`, `to` and `subject` fields perform [string substitution](#string-substitution-syntax).
```
Comment (fgalan) [inline documentation/plain_rules.md]: I understand that to keep documentation coherent, the list just under "This substitution can be used in the the following fields" at "String substitution syntax" should be also modified.

Comment (crbrox) [inline documentation/plain_rules.md]: Fix fbcabc8

File: documentation/plain_rules.md (hunk 3 above)
Comment (crbrox) [inline documentation/plain_rules.md]: Fixed fbcabc8

File: documentation/plain_rules.md (hunk 2 above)
Comment (crbrox) [inline documentation/plain_rules.md]: Fixed fbcabc8

File: CHANGES_NEXT_RELEASE (hunk 1 above)
Comment (crbrox) [inline CHANGES_NEXT_RELEASE]: Fixed a20d23c

**Comment by fgalan** [comment]
LGTM (although it seems to be failing in Travis...)

Passing the ball to @dmoranj for final LGTM + merge + delete branch

**Comment by crbrox** [comment]
Fixing travis ... not to be merged yet

**Comment by crbrox** [comment]
Fixed travis at 091c7e7

**Comment by dmoranj** [comment]
LGTM

**Pull request merged**